src/python/dxpy/dx_extract_utils/ExpressionInputsValidator.py

```diff
@@ -0,0 +1,164 @@
+from ..exceptions import err_exit
```

Comment:
This is not needed here, it can be passed directly to the `input_validator = ExpressionInputsValidator` call in `src/python/dxpy/cli/dataset_utilities.py`